Makefile

```diff
@@ -413,13 +413,13 @@ prepare-coverage-test: $(GOTESTSUM) $(TEST_OUTPUT_ROOT)
 
 unit-test-coverage: prepare-coverage-test
 	@printf $(COLOR) "Run unit tests with coverage..."
-	go run ./cmd/tools/test-runner $(GOTESTSUM) -retries $(FAILED_TEST_RETRIES) --junitfile $(NEW_REPORT) --packages $(UNIT_TEST_DIRS) -- \
+	go run ./cmd/tools/test-runner $(GOTESTSUM) -retries $(FAILED_TEST_RETRIES) --junitfile $(NEW_REPORT) --packages="$(UNIT_TEST_DIRS)" -- \
```

Comment:
Are you sure you need `=` here?

Reply:
No, but it aligns with the change I'm making in the code coverage PR (where I'm changing all of them to use `=`)

Reply:
Add them for other flags on this line then.

Reply:
I can't yet, because testrunner expects them to be like this right now.